### ISSUE TITLE
Move away from `#FFFFFF` (pure white) colours on dark mode

### ISSUE THREAD

**Issue by ShadowJonathan** [vector-im/element-web#18116]
(Originally I was going to comment on https://github.com/matrix-org/matrix-react-sdk/pull/5672 for this, and mention that the context menus feel too overpowering for some reason, and was about to suggest changing to `#EEEEEE` colours for icons and text, but then i realised this applied to Element in general.)

#### Is your suggestion related to a problem? Please describe.
Currently, the background for Element on dark mode is `#15191e`, while the text is `#ffffff`, that is effectively full white on black.

Coming from discord, this look was initially a bit jarring to be, but I got used to it and ignored the harshness.

However, recently with the change of https://github.com/matrix-org/matrix-react-sdk/pull/5672, I realised this was an actual problem, and I think it'd be much softer on the eyes to change the default dark mode text colour from `#FFFFFF` to something softer (`#EEEEEE` or `#E0E0E0` or the likes).

This'd help with reading Element conversations late at night, and would also help with above context menu change, to have it feel less intensive.

In other words, if i were to look at a conversation late at night, i'd rather have it be something like this;

<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/22740616/126363740-0d476c8d-8786-468a-bda2-70460246e663.png)

</details>

Over this;

<details>
<summary>Image</summary>

![image](https://user-images.githubusercontent.com/22740616/126363975-82d97d63-f858-4980-842e-7dcfb867faaa.png)

</details>

As the former has less contrast, and less "harshness" to the eyes.

**Comment by robintown** [comment]
Mostly a duplicate of https://github.com/vector-im/element-web/issues/8664

**Comment by ShadowJonathan** [comment]
I'd be willing to close this one if some updates/reactions from design/devs could be made here/there, as mostly I'm just concerned if that issue is still noticed, and if this is a concern of Design at all (and therefore on the backlog).

If it is on the backlog, i'll close this and move insights to that issue :+1: I just wanted to give it a little bump.

**Comment by turt2live** [comment]
Considering this a duplicate of https://github.com/vector-im/element-web/issues/8664
